src/EnergyPlus/HVACInterfaceManager.hh

```diff
@@ -160,20 +146,12 @@ struct HVACInterfaceManagerData : BaseGlobalStruct
 
     bool CommonPipeSetupFinished = false;
     Array1D<HVACInterfaceManager::CommonPipeData> PlantCommonPipe;
-    Array1D_bool MyEnvrnFlag_SingleCommonPipe;
-    Array1D_bool MyEnvrnFlag_TwoWayCommonPipe;
-    bool OneTimeData_SingleCommonPipe = true;
-    bool OneTimeData_TwoWayCommonPipe = true;
```

Comment:
Removed these four variables from state, and from having to be allocated/checked later.  This little state struct should continue to be pruned and deleted.